xdsl/dialects/csl.py

```diff
@@ -376,6 +376,13 @@ class ImportModuleConstOp(IRDLOperation):
 
     result = result_def(ImportedModuleType)
 
+    def __init__(self, name: str, *params: SSAValue | Operation):
+        super().__init__(
+            operands=[[]] if len(params) == 0 else params,
```

Comment:
Two things:

1. Params should be a single `SSAValue | Operation | None = None` operand, as `import_module` only takes a single argument 
2. I'm 99% sure that you can just write ` operands=[param]` (in both cases where param is a vararg, and if it's optional)

Reply:
changing as suggested